.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+# Invoke jobs via workflows
+# See: https://circleci.com/docs/2.0/configuration-reference/#workflows
+workflows:
+  build-and-notify:
+    jobs:
+        - build:
```

Comment:
we need to define this job, above , here in the workflow you are just "calling" a build job, which the CI file does not know anything about